src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1363,6 +1363,9 @@ MavlinkReceiver::handle_message_set_attitude_target(mavlink_message_t *msg)
 			_control_mode_sub.copy(&control_mode);
 
 			if (control_mode.flag_control_offboard_enabled) {
+				if (_vehicle_status_sub.updated()) {
```

Comment:
Can you get rid of `_vehicle_status` and just do the copy every time? The copy is pretty cheap and most of the time `_vehicle_status` isn't used.

Reply:
Why not :)